packages/g/gum/package.yml

```diff
@@ -0,0 +1,33 @@
+name       : gum
+version    : 0.13.0
+release    : 1
+source     :
+    - https://github.com/charmbracelet/gum/archive/refs/tags/v0.13.0.tar.gz : 329a38f3453b4be1f00e1fcb987aacf574fe3a8cc592084529c05716ddf4e7c4
+homepage   : https://charm.sh
+license    : MIT
+component  : programming.tools
+summary    : A tool for glamorous shell scripts
+description: |
+  Effects include
+  choose     Choose an option from a list of choices
+  confirm    Ask a user to confirm an action
+  file       Pick a file from a folder
+  filter     Filter items from a list
+  format     Format a string using a template
+  input      Prompt for some input
+  join       Join text vertically or horizontally
+  pager      Scroll through a file
+  spin       Display spinner while running a command
+  style      Apply coloring, borders, spacing to text
+  table      Render a table of data
+  write      Prompt for long-form text
+  log        Log messages to output
+networking : yes
+builddeps  :
+    - golang
+build      : |
+    go build -v
+install    : |
+    install -Dm00755 gum $installdir/usr/bin/gum
+    install -Dm00644 README.md $installdir/usr/share/docs/gum/README.md
+    install -Dm00644 LICENSE $installdir/usr/share/docs/gum/LICENSE
```

Comment:
We don't usually include license files in the package since the license is already in the metadata of the package.